### DISCUSSION TITLE
How to disable the confirmation alert message?

### DISCUSSION THREAD

**Discussion by aptash** [NightscoutFoundation/xDrip#3200] [Q&A]
Hello!
Is there any way to disable this confirmation alert message?
![image](https://github.com/NightscoutFoundation/xDrip/assets/5793551/b6e3f610-c6d9-4fbc-922e-6c9a5c04ca23)

I have a case when I use MacroDroid to clean the xDrip notification about "BG not available" and do some work after that. After these actions I have this alert in xDrip. Also these alerts are collected so I have to press many times to close them all.
Thanks in advance!

**Comment by Navid200** [comment]
I only get that confirmation request if I snooze an alert immediately after an alert triggers.
On average, I estimate I get it once every 3 months.

Please explain your scenario and why you use Macrodroid.
Basically, please help us see why this is a common problem that can annoy many users under normal use case.
On the other hand, if this is an extremely special scenario, hopefully, you can figure out a workaround to avoid it.

**Reply by aptash** [reply]
First of all,  I don't think it's a common problem.

My case:
I use Sibionics CGM and the patched app SugarJoy+ to send the data to xDrip.
Sometime SugarJoy+ (SJ+) can't connect to the sensor. And I need manually to press the button "Connect" in SJ+.

So I made the following scenario in MacroDroid (MD) when the screen is black:
1. MD listens to xDrip notification about "BG not available"
2. MD cleans this xDrip notification - _after this step I get the confirmation alert message in xDrip_
3. MD forcefully start SJ+ - _this step helps to restore connection to the sensor_
4. MD opens main screen - _so I don't see SJ+ each time when I unlock the phone_
5. MD sends the note to NightScout about this event via http-request

xDrip settings are like this: 
throw an alert about "BG not available" after 6 minutes and repeat it every 1 minute.

Will it help if I clean the xDrip notification not immediately but after a few seconds?

P.S.: some people use more simple scenario in MacroDroid:
when the screen is black, MD forcefully start SJ+ every 5 minutes.
So they don't listen to any xDrip notifications.
